src/components/omnibox/omnibox.css

```diff
@@ -1,3 +1,15 @@
+:root {
```

Comment:
This is a big ask, but would it be possible to consolidate these light/dark theme colors into a single file? Sort of like Material Design's color system, with `colorPrimary`, `colorSecondary`, `colorBackground`, etc., to reduce duplication. This is our fault for not doing this in the first place, so we can always create a new issue and address it separately.